WorkloadManagementSystem/scripts/dirac-jobexec.py

```diff
@@ -54,6 +54,13 @@ def jobexec( jobxml, wfParameters ):
   for pName, pValue in wfParameters.items():
     workflow.setValue( pName, pValue )
 
+  # Propagate the command line parameters to the workflow module instances of each step
+  for _stepName, stepdefinition in workflow.step_definitions.iteritems():
```

Comment:
`itervalues()` instead of `iteritems()`?

Reply:
Done